vizro-core/changelog.d/20240305_142454_huong_li_nguyen_fix_card_validation.md

```diff
@@ -0,0 +1,46 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Changed
+
+- Allow `Page.title` to be inserted directly into `Card.href` as a relative link. ([#347](https://github.com/mckinsey/vizro/pull/347))
+
+<!--
+### Deprecated
+
+- A bullet item for the Deprecated category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Fixed
+
+- Align validators for `Card.href` and `Page.path` to ensure consistent navigation linking. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

Comment:
```suggestion
- Fix [bug](https://github.com/mckinsey/vizro/issues/285) which breaks page opening when providing the same `str to `Card.href` and `Page.path`, when that `str` is not formatted as a valid `path`. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

I think it would be advantageous to communicate the bug then, and give an idea if the user would be affected. Previously I have to guess what this really means.

Overall I would suggest linking the bug in the PR description, and labelling the PR as such as I wasn't aware this was a bug even.

Reply:
I can add it to the PR description as well 👍 

However, the PR was already labeled as a bug and the issue ticket was also connected before such that it automatically closes the issue if this PR gets merged - you can see it in the development section of this PR! :) 

![Screenshot 2024-03-06 at 09 19 06](https://github.com/mckinsey/vizro/assets/90609403/b657cc80-5587-4d23-8685-1f02deb7f347)

Reply:
Ah mea culpa 🙏 I find it always weirdly hidden on the right there haha. Awesome!

Reply:
I've rephrased a bit to make it shorter, but kept your message:

Fix [bug](https://github.com/mckinsey/vizro/issues/285) where navigating from a `Card` to `Page` failed when an unformatted `Page.path` was used in `Card.href`. ([#347](https://github.com/mckinsey/vizro/pull/347))